.github/workflows/simulation_test.yml

```diff
@@ -201,6 +234,47 @@ jobs:
         mpirun -np 2 special -mpi -python $NEURODAMUS_PYTHON/init.py --configFile=simulation_sonata_coreneuron.json
         ls reporting_coreneuron/*.h5
 
-    # - name: live debug session, comment out
-    #   if: failure()
-    #   uses: mxschmitt/action-tmate@v3
+    - name: Install test and doc dependencies
+      run: |
+        pip install -r tests/requirements.txt
+        pip install "sphinx<5.1.0" sphinx-bluebrain-theme
+
+    - name: test-unit
+      run: |
+        pytest tests/unit
+
+    - name: test-integration
+      run: |
+        pytest -x --forked tests/integration
+
+    - name: test-integration-e2e
+      run: |
+        pytest -s -x --forked --durations=5 --durations-min=15 tests/integration-e2e
+
+    - name: test-scientific
+      run: |
+        pytest -s -x --forked --durations=5 --durations-min=15 tests/scientific
+
+    - name: test-scientific-ngv
+      run: |
+        export HOC_LIBRARY_PATH=$NEURODAMUS_NEOCORTEX_MULTISCALE_ROOT/share/neurodamus_neocortex/hoc
+        export NRNMECH_LIB_PATH=$NEURODAMUS_NEOCORTEX_MULTISCALE_ROOT/lib/libnrnmech.so
+        unset CORENEURONLIB
+        pytest -s -x --forked --durations=5 --durations-min=15 tests/scientific-ngv
+
+    - name: Build docs
+      run: |
+        export PIP_INDEX_URL=https://bbpteam.epfl.ch/repository/devpi/simple
+        sphinx-build -T -W docs docs/_build
```

Comment:
`PIP_INDEX_URL` should not be accessible outside the VPN, is that required?

Also, do we need to build the docs for all Python versions? Kind of wondering if we could factor that out into a separate workflow.

Reply:
Yes, just one version is appropriate. Come to think of it, is that duplicating the readthedocs build?